pkg/utils/utils.go

```diff
@@ -0,0 +1,16 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package utils
+
+import "encoding/json"
+
+// JSONMustMarshal marshals the input to JSON []byte and panics if it fails.
+func JSONMustMarshal(input interface{}) []byte {
+	res, err := json.Marshal(input)
+	if err != nil {
+		panic(err)
```

Comment:
panic looks pretty extreme here at the same time we should be in full control for all the messages that are converted ...

Reply:
The `JSONMustMarshal` is only used for global error variables where the client sends back JSON back to the agent. So if they failed it would be on startup and during the unit tests.